src/router/index.ts

```diff
@@ -4,6 +4,7 @@ import HomeView from '../views/HomeView.vue'
 import MainGameView from '../views/MainGameView.vue'
```

Comment:
Please use @ for imports, like "import MainGameView from '@/views/MainGameView.vue'"